### ISSUE TITLE
Verifying Deployment fails when using docker.networks

### ISSUE THREAD

**Issue by mfruhner** [zodern/meteor-up#1096]
Hello,
I encounter a bug, where my app won't update, because the deployment verification fails and the app is rolled back to the previous version.
This only happens when I specify an additional network, which I need the app to talk to.
If I remove the networks array the deployment verification succeeds and the app is updated correctly. When I re-add the network and cancel the verification (Ctrl+C), it does not try to roll back and the updated version works perfectly fine on the server and even the network connection is created.

If you take a look at the output, it shows two IP addressess concatinated and the message "could not resolve host". Which is obvious, if it actually tries to connect to this address. Maybe its just an output bug, though.

Any thoughts?

Thanks,
Maik

Mup version (`mup --version`):
1.4.6

Mup config

```js
{
  "servers": {
    "one": {
      "host": "1.2.3.4",
      "username": "maik",
      "password": "password"
    }
  },
  "app": {
    "name": "my-app",
    "path": "../",
    "servers": {
      "one": {}
    },
    "buildOptions": {
      "serverOnly": true
    },
    "env": {
      "ROOT_URL": "http://subdomain.subdomain.host.com",
      "MONGO_URL": "mongodb://mongodb:27017/my-app",
      "MONGO_OPLOG_URL": "mongodb://mongodb/local"
    },
    "docker": {
      "image": "abernix/meteord:node-8.4.0-base",
      "prepareBundle": true,
      "networks": [
        "msnetwork"
      ],
      "stopAppDuringPrepareBundle": true,
      "args": [
        "--link=mongodb:mongodb"
      ]
    },
    "enableUploadProgressBar": true,
    "type": "meteor"
  },
  "mongo": {
    "version": "3.4.1",
    "servers": {
      "one": {}
    }
  }
}
✓ Config is valid
```

Output of command

```bash
Verifying Deployment
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: 172.17.0.3172.18.0.3
```

**Comment by mfruhner** [comment]
So i found the error.
In `/src/plugins/meteor/assets/meteor-deploy-check.sh` the IP address of the docker container is fetched in an old deprecated way:

`CONTAINER_IP=$(docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $APPNAME)`

This prints all network IP in a row.

As a comment to the solution of [this ](https://stackoverflow.com/questions/17157721/how-to-get-a-docker-containers-ip-address-from-the-host) SO question states, the new syntax is:

`docker inspect --format='{{ .NetworkSettings.Networks.bridge.IPAddress }}'`

(when using the bridge network setting)
This only prints the first IP address.

Shoud I create a merge request for this or can you update it?


Edit:
Just tested this by changing this line in my local mup installation and running a `mup reconfig`.
The deployment verification works like a charm!

Edit 2:
Just found  #953 which already fixed this. But it uses more like a workaround rather then the new recommended way of using the docker command.

**Comment by zodern** [comment]
This should be fixed in Mup 1.5.
